RELEASE.adoc

```diff
@@ -13,25 +13,40 @@ The publication is decoupled. See link:https://github.com/eclipse-che/che-docs/t
 
 .Prerequisites
 
-* Define the Eclipse Che version to release: `__<major>__.__<minor>__.0`.
+* The new release is published on https://github.com/eclipse/che/releases and https://mattermost.eclipse.org/eclipse/channels/eclipse-che-releases.
 
 .Procedure
 
-. Create the `__<major>__.__<minor>__.x` branch from the prerelease branch.
+. From the `main` branch, create a release branch with a branch name in the format `__<major>__.__<minor>__.x`, and then checkout the created release branch.
++
+NOTE: The name of the release branch differs from the number of the release in the trailing `x`.
 
-. Edit the `antora.yml` file in the `__<major>__.__<minor>__.x` branch to define:
+. In the `__<major>__.__<minor>__.x` branch, update the following product-version-related values in `antora.yml`:
```

Comment:
I tried doing this while in `main`, but that requires a PR because `main` appears to be protected.
I'm adding a separate step for updating the same values for `main`.